src/components/common/StagingArea/StagingAreaSetting.tsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+import ArrangeBtn from '../arrangeBtn/ArrangeBtn';
+import TextBtn from '../button/textBtn/TextBtn';
+
+interface StagingAreaSettingProps {
+	// 전체 / 지연인지 클릭이벤트?
+}
+
+function StagingAreaSetting(props: StagingAreaSettingProps) {
+	const [activeButton, setActiveButton] = useState<'전체' | '취소'>('전체');
+
+	const handleButtonClick = (button: '전체' | '취소') => {
+		setActiveButton(button);
+	};
+
+	return (
+		<StagingAreaSettingLayout>
+			<TextBtnContainer>
+				<TextBtn
+					size="small"
+					text="전체"
+					color={activeButton === '전체' ? 'BLUE' : 'WHITE'}
+					mode={activeButton === '전체' ? 'DEFAULT' : 'LIGHT'}
+					isHover
+					isPressed
+					onClick={() => handleButtonClick('전체')}
+				/>
+				<TextBtn
+					size="small"
+					text="취소"
+					color={activeButton === '취소' ? 'BLUE' : 'WHITE'}
+					mode={activeButton === '취소' ? 'DEFAULT' : 'LIGHT'}
+					isHover
+					isPressed
+					onClick={() => handleButtonClick('취소')}
+				/>
```

Comment:
```suggestion
				{(['전체', '취소'] as const).map((button) => (
					<TextBtn
						key={button}
						size="small"
						text={button}
						color={activeButton === button ? 'BLUE' : 'WHITE'}
						mode={activeButton === button ? 'DEFAULT' : 'LIGHT'}
						isHover
						isPressed
						onClick={() => setActiveButton(button)}
					/>
				))}
```

여길 이렇게 써보는 방식은 어떨까요 ??
반복되는 코드들도 사라지고 `handleButtonClick` 도 사라져도 되니 더 깔끔할 것 같습니다!

Reply:
헉 짱짱 바로 반영할게요 `map`을 생각을 못했다!!